example/public/index.html

```diff
@@ -1,49 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
 
-<head>
-    <meta charset="utf-8">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
-    <!--
-        [ REQUIREMENT ]
-        Loads the polyfills according to the user browser. Used mainly to run
-        this demo on IE11
-    -->
-    <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
-
-    <!--
-        [ REQUIREMENT ]
-        This script needs to be included only for IE11 to make css variables alive.
-        Mind that it needs to be loaded in a blocking way (without async or defer)
-        in order to prevent style flickering
-    -->
-    <script src="https://unpkg.com/css-vars-ponyfill@1"></script>
-    <script type="text/javascript" src="/css-vars-config.js"></script>
-
-    <!-- Other dependencies, such as fonts -->
-    <link href="https://fonts.googleapis.com/css?family=Charm" rel="stylesheet">
-
-    <!--
-        [ REQUIREMENT ]
-        Loads the `OneUI CSS` with its own variables.
-        The theme must go after in order to be able to override variables
-    -->
-    <link data-oneui-level="oneui" rel="stylesheet" href="./oneui.min.css">
-
-
-    <!-- Theme that comes from a CDN (in this app this is apply progrematically: see App.js:toggleTheme) -->
-    <!-- <link data-oneui-level="theme" rel="stylesheet" href="./theme-custom.css"> -->
-
-    <title>OneUI Themable Example</title>
-
-</head>
-
-<body>
-
-    <div id="root"></div>
-
-</body>
-
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="utf-8">
+        <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <link href="https://fonts.googleapis.com/css?family=Charm" rel="stylesheet">
+
+        <!--
+            [ REQUIREMENT ]
+            Loads the polyfills according to the user browser. Used mainly to run
+            this demo on IE11
+        -->
+        <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
```

Comment:
So this one is not something mandatory for a project? Just for demo purposes?

Reply:
Is mandatory to run the demo on IE, since it doesn't support Promises. But in modern browsers it's not required.

Reply:
So would it still be loaded even if you use a modern browser?

Reply:
This exact polyfill actually is pretty smart. It detects a browser version by a user agent string and returns only necessary javascript:

![image](https://user-images.githubusercontent.com/403250/52471354-feb3ec00-2b90-11e9-82f8-1d45be0dc944.png)

But I believe that deal with polyfills is a responsibility of an app/product. We just need to declare required tech we rely on.